src/components/ATATSlideoutPanel.vue

```diff
@@ -48,6 +48,9 @@ import SlideoutPanel from "@/store/slideoutPanel/index";
 
 export default class ATATSlideoutPanel extends Vue {
   @Prop({ default: "380" }) private panelWidth!: string;
+  @Prop({ default: false }) private alwaysOpen!: boolean;
+  @Prop({ default: true }) private showHeader!: boolean;
```

Comment:
Don't need this prop - just don't pass a `title` prop to ATATSlideoutPanel component if don't want to show the "header"/title area

Reply:
removed

Reply:
Done